src/tutorials/0004-mutable-file-system/08.md

```diff
@@ -8,7 +8,7 @@ MFS allows you to move files between directories just as you would on your local
 The method looks like this:
 
 ```js
-await ipfs.files.mv(...from, to, [options])
+await ipfs.files.mv(from, to, [options])
```

Comment:
```suggestion
await ipfs.files.mv(...from, to, [options])
```

Probably best to match the way the API docs describe this, but I've suggested tweaks to your text below to accommodate.